### ISSUE TITLE
Support for ppc64le official image

### ISSUE THREAD

**Issue by vaibhavsood** [influxdata/influxdata-docker#168]
Hi, want to check if ppc64le official docker images are supported for influxdb? I could not find it listed as an architecture https://hub.docker.com/_/influxdb/

Thanks!

**Comment by jsternberg** [comment]
I don't think we build software for that architecture so we wouldn't be able to build docker images for it. I'll keep this open to remind me that this exists and so maybe we can start building for that architecture, but for now, you'll probably have to figure out how to compile it from source and build the images yourself.

For anyone who needs this and wants it officially supported, please add a 👍 to the top comment so we can track interest for this architecture.
